labcontrol/db/process.py

```diff
@@ -3623,7 +3629,8 @@ def _generate_amplicon_prep_information(self):
             for result in TRN.execute_fetchindex():
                 result = dict(result)
                 study_id = result.pop('study_id')
-                sid = result.pop('sample_id')
+                # sid unused when generating merged sample/controls prep file
+                # sid = result.pop('sample_id')
```

Comment:
Can this be removed?

Reply:
Nice catch! I wonder if that came back in with the auto-merge; one moment...